python/grass/jupyter/utils.py

```diff
@@ -202,6 +202,16 @@ def get_rendering_size(region, width, height, default_width=600, default_height=
     return (default_width, round(default_width * region_height / region_width))
 
 
+def save_vector(geojson_filename, name):
```

Comment:
This is rather trivial function, perhaps this could include the temporary file creation (and removal).

Reply:
To clarify, should I add a button to delete the the drawn vector?

Reply:
I am talking about moving some of the code handling temporary file creation to this function. The temporary file needs to be removed once it's not needed.